src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -102,28 +102,32 @@ export default function DocumentList({
               </span>
               <span className="label col-sm-4 d-lg-none">Action (s):</span>
               <span className="col-sm-8 col-lg-2 file-cell">
-                <span
-                  id={`withdraw_${document.identifier}`}
-                  className="file-href"
-                  role="button"
-                  tabIndex={0}
-                  onClick={(e) => handleWithdrawFileEvent(e, document)}
-                  onKeyDown={(e) => handleWithdrawFileEvent(e, document)}
-                >
-                  withdraw
-                </span>
-                <MdHelp
-                  style={{
-                    fontSize: "20px",
-                    marginLeft: "10px",
-                    color: "#7598ca",
-                  }}
-                  id="withdraw-tooltip"
-                />
-                <BcGovTooltip
-                  target={`withdraw-tooltip`}
-                  content={withdrawTooltipData}
-                />
+                {!document.status.description.match(/withdrawn/i) && (
```

Comment:
Is there a status code value to match on instead of matching on the status description?